### PR TITLE
Fixing tests for accessing data from xcube server via python api

### PR DESCRIPTION
**Pull request by AliceBalfanz**
This PR only fixes the tests for test_s3buckethandlers.py when serving data the classical way: one dataset per identifyer, as in `examples/serve/demo/config.yml` . The data access does not work yet for cubes served via DataStores like in the config `examples/serve/demo/config-with-stores.yml`
Checklist:

* [ ] ~~Add unit tests and/or doctests in docstrings~~
* [ ] ~~Add docstrings and API docs for any new/modified user-facing classes and functions~~
* [ ] ~~New/modified features documented in `docs/source/*`~~
* [ ] ~~Changes documented in `CHANGES.md`~~
* [x] AppVeyor CI passes
* [x] Test coverage remains or increases (target 100%)

Remember to close associated issues after merge!

### CODE REVIEW

**Review by TonioF — COMMENTED** [review]
I tried to review by following the steps to reproduce in the original issue. I could not reproduce it. The tests work your changes, too, though.

**Comment by AliceBalfanz** [comment]
> I tried to review by following the steps to reproduce in the original issue. I could not reproduce it. The tests work your changes, too, though.

If you delete your .aws (or give it a different name) then you will see the error with the old code :)

**Review by TonioF — APPROVED** [review]
Okay, I could test it now. Works fine!

File: test/webapi/test_s3buckethandlers.py [hunk 1]
```diff
@@ -31,13 +31,20 @@ class S3BucketHandlersTest(unittest.TestCase):
...
 
     @unittest.skipUnless(XCUBE_SERVER_IS_RUNNING, SKIP_HELP)
     def test_open_cube_from_xube_server_abs_path(self):
         ds = open_cube('http://localhost:8080/s3bucket/local',
-                       format_name='zarr')
+                       format_name='zarr',
+                       s3_kwargs={
+                           'anon': True
+                       }
+                       )
```
Comment (forman) [inline test/webapi/test_s3buckethandlers.py:47]:
```suggestion
                       })
```

**Review by forman — CHANGES_REQUESTED** [review]
Just tiny minor formatting issue :)

**Pull request merged**